string-manipulation-1/src/capitalize.js

```diff
@@ -0,0 +1,15 @@
+/* exported capitalize */
+/* - create a storage space and store the first letter of each word and uppercase them
```

Comment:
In a lot of your pseudocode, you are using the word "each" often as if referring to multiple uses of this function, but the pseudocode should only be phrased as the plan for a single use. For instance here, this should be phrased as "create a storage space and store the first letter of the given word and uppercase it."

Reply:
Make sure to adjust this in your other pseudocode as necessary, too 👀